docs/faults.rst

```diff
@@ -0,0 +1,32 @@
+View Faults (Supported in v2.4+)
+===============================================
+
+View App Faults
+---------------
+
+1. Traverse to top-right corner menu.
+
+2. Click :guilabel:`App Faults`.
```

Comment:
I'd combine steps one and two

1.  From the top-right corner menu, click :guilabel:`App Faults`.